enterprise/server/byte_stream_server_proxy/byte_stream_server_proxy.go

```diff
@@ -47,16 +53,23 @@ func New(env environment.Env) (*ByteStreamServerProxy, error) {
 		return nil, fmt.Errorf("A remote ByteStreamClient is required to enable ByteStreamServerProxy")
 	}
 	return &ByteStreamServerProxy{
-		atimeUpdater: atimeUpdater,
-		local:        local,
-		remote:       remote,
+		atimeUpdater:  atimeUpdater,
+		authenticator: authenticator,
+		local:         local,
+		remote:        remote,
 	}, nil
 }
 
 func (s *ByteStreamServerProxy) Read(req *bspb.ReadRequest, stream bspb.ByteStream_ReadServer) error {
 	ctx, spn := tracing.StartSpan(stream.Context())
 	defer spn.End()
 
+	if authutil.EncryptionEnabled(ctx, s.authenticator) {
+		metrics.ByteStreamProxyReads.With(
+			prometheus.Labels{metrics.CacheHitMissStatus: "miss"}).Inc()
```

Comment:
do you want a separate label here? it might throw off the cache hit stats if encrypted data is always counted as a miss.